Sources/GRPCCodeGen/Internal/Translator/ClientCodeTranslator.swift

```diff
@@ -267,7 +267,7 @@ extension ClientCodeTranslator {
     in service: CodeGenerationRequest.ServiceDescriptor
   ) -> ParameterDescription {
     let requestType = method.isInputStreaming ? "Stream" : "Single"
-    let clientRequestType = ExistingTypeDescription.member(["ClientRequest", requestType])
+    let clientRequestType = ExistingTypeDescription.member(["GRPCCore.ClientRequest", requestType])
```

Comment:
Same here (and in a bunch of places), `GRPCCore` should be an additional string in the array